test/es-module/test-import-empty.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { spawnPromisified } = require('../common');
+const fixtures = require('../common/fixtures.js');
+const assert = require('node:assert');
+const { execPath } = require('node:process');
+const { describe, it } = require('node:test');
+
+describe('Import empty module', { concurrency: true }, () => {
+  it(async () => {
+    const { code, signal, stdout, stderr } = await spawnPromisified(execPath, [
+      '--no-warnings',
+      '--eval',
+      'import("empty")',
+    ], {
+      cwd: fixtures.path(),
+    });
+    assert.strictEqual(code, 1);
+    assert.strictEqual(signal, null);
+    assert.strictEqual(stdout, '');
+    assert.match(stderr, /No package entry point defined for package/);
```

Comment:
Please reorder them, so we can get a more useful error if the test ever fails ("0 is not strictly equal to 1" is not a very useful error)
```suggestion
    assert.match(stderr, /No package entry point defined for package/);
    assert.strictEqual(stdout, '');
    assert.strictEqual(code, 1);
    assert.strictEqual(signal, null);
```